miso-web/src/main/webapp/scripts/hot_tissuepiecetype.js

```diff
@@ -26,7 +26,7 @@ HotTarget.tissuepiecetype = {
     return !config.isAdmin ? [] : [{
       name: 'Edit',
       action: function(items) {
-        window.location = window.location.origin + '/miso/tissuepiecetype/bulk/edit?' + jQuery.param({
+        window.location = Uwindow.location.origin + Urls.tissuePieceTypes.bulkEdit + '?' + jQuery.param({
```

Comment:
Can keep or remove the `window.location.origin`, but it looks like a stray `U` got in there